packages/playground-ui/src/index.css

```
@@ -61,6 +61,18 @@
   --warning1: var(--accent6);
   --positive1: var(--accent1);
   --negative1: var(--accent2);
+
+  /* Notice admonition colors — bg uses /20 alpha overlay; fg is the on-surface text/icon color */
+  --notice-success: oklch(0.92 0.27 138);
```

Stylelint (17.9.0)

[error] Expected "138" to be "138deg" (hue-degree-notation)

(hue-degree-notation)